portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -4,40 +4,48 @@
 
 /** Represents a comment with related details. */
 public final class Comment implements EntityConvertible {
+    private static final String KIND = "Comment";
+    private static final String PROPERTY_NAME_COMMENTER = "commenter";
+    private static final String PROPERTY_NAME_CONTENT = "content";
+    private static final String PROPERTY_NAME_TIMESTAMP = "timestamp";
+
+    private final long id; // for json creation
     private final String commenter;
     private final String content;
     private final long timestamp;
 
     public Comment(String commenter, String content, long timestamp) {
+        this.id = 0;
         this.commenter = commenter;
         this.content = content;
         this.timestamp = timestamp;
     }
 
-    static class PropertyName {
-        private final static String KIND = "Comment";
-        private final static String COMMENTER = "commenter";
-        private final static String CONTENT = "content";
-        private final static String TIMESTAMP = "timestamp";
+    public Comment(long id, String commenter, String content, long timestamp) {
```

Comment:
Mark this private for now as it's only meant to be used in this class?

Reply:
Make sense

Reply:
Fixed